.babelrc

```diff
@@ -1,19 +1,22 @@
 {
   "presets": [
-    "env",
-    "react"
+    "@babel/preset-env",
+    "@babel/preset-react"
   ],
   "plugins": [
-    "transform-object-rest-spread",
-    "transform-class-properties",
-    "transform-runtime"
+    "@babel/plugin-proposal-object-rest-spread",
+    "@babel/plugin-proposal-class-properties",
+    "@babel/plugin-transform-runtime"
   ],
   "env": {
     "test": {
       "presets": [
-        "es2015",
-        "./babel"
+        ["./babel", {
```

Comment:
So we're on the same page, I'm a little confused about the expectations you have for this vs the presets you have in the top-level `presets` array. As it is, this `test` env will just run _both_ sets of presets. What specifically is the goal of the `test` env here? It'll add a few extra plugins around styling and react it looks like, but it also doubles up on a _ton_ of the stuff you're already specifying in the top level config.

Reply:
Right, this should probably be broken out into `.babelrc` in the `test` directory. Will do that after this PR lands. Since it's not directly related to babel 7 👍